docs/content/docs/connectors/table/pulsar.md

```diff
@@ -56,6 +56,8 @@ This table outlines options for the Pulsar SQL connector.
 | `source.stop.after-message-id` | (none)                     | String             | (Optional) message id used to specify a stop position but include the given message in the consuming result for the unbounded sql source. Pass in a message id representation in `ledgerId:entryId:partitionId`, such as `12:2:-1`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
 | `topics`                       | (none)                     | List&lt;String&gt; | Topic name(s) the table reads data from. It can be a single topic name or a list of topic names separated by a semicolon symbol (`;`) like `topic-1;topic-2`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
 
+However, besides these Pulsar SQL connector options, you can configure the underlying Pulsar DataStream Connector using the connector options defined in [Apache Pulsar Source Connector](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/pulsar/#source-configurable-options) and [Apache Pulsar Sink Connector]([subscription type]([Apache Pulsar Source Connector](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/pulsar/#sink-configurable-options) using `WITH` clause.
```

Comment:
Use the inline document link instead of hard code URL.

Reply:
This documentation is copied from the SN hub documentation. so to keep it in sync with the SN hub doc, we need to use a hard code url

Reply:
If the content is going to be added to the Flink community, you can use the internal link. Because the linked content has existed in the Flink community. If we want to use them in SN Hub, we need to use an external link.

Reply:
This content is not going to be added Flink community recently. the rationale behind I use hard code url in this documentation is :

1. This documentation is meant to be hidden from user, users should use the SN hub version as the single source of truth for now.
2. The documentation is kept so everytime we commit a change related to SQL connector, we need to update the documentation as well.
3. In each release, we just copy this documentation and replace the corresponding part in pulsar-hub documentation.


When we finally decide to contribute the Flink community, I plan to atomically
1. replace all hard url with project relative path
2. merge to Flink community
3. rewrite SN hub documentation